sql-cli/noxfile.py

```diff
@@ -25,6 +25,12 @@ def dev(session: nox.Session) -> None:
 def test(session: nox.Session, airflow: str) -> None:
     """Run both unit and integration tests."""
 
+    # Airflow automatically creates the ~/airflow directory on importing airflow
+    # but we are not using this directory hence we change the home to a tmp directory to avoid side effects.
+    env = {
+        "AIRFLOW_HOME": session.create_tmp(),
+    }
```

Comment:
```suggestion
    env = {
        # Airflow automatically creates the ~/airflow directory on importing airflow
        # but we are not using this directory hence we change the home to a tmp directory to avoid side effects.
        "AIRFLOW_HOME": session.create_tmp(),
    }
```